### PR TITLE
Lmb 420/entity to add email to bestuursorgaan

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## Description

We want every bestuurseenheid to add an email adress as contact information. The make this possible anew page `/settings` is created where they can fill in a form. This form will update the bestuurseenheid-contact model with the inserted email adres.

## How to test

1. In the topbar you find the bestuurseenheid name
2. This is a dropdown where the `Instellingen` is added
3. When routing through `/settings` a form is shown where the user can input the e-mailadres

## Notes
- now when you confirmed the changes in the form the button will show a loading message 
    -  the modal closes after it has saved the changes

## Attachments

image

image

## Links to other PR's

- https://github.com/lblod/app-lokaal-mandatenbeheer/pull/133

### CODE REVIEW

File: app/components/form/save-with-history.hbs [hunk 1]
```diff
@@ -17,7 +17,11 @@
   <Modal.Footer>
     <AuToolbar as |Group|>
       <Group>
-        <AuButton {{on "click" @save}}>
+        <AuButton
+          {{on "click" @save}}
+          @loading={{@isSaving}}
+          @loadingMessage="Aan het opslaan"
+        >
```
Comment (JonasVanHoof) [inline app/components/form/save-with-history.hbs:24]: Not PR related but added this as when you are waiting on the save nothing is shown to the user

File: app/components/form/instance.hbs [hunk 2]
```diff
@@ -31,6 +31,7 @@
           >Pas aan</AuButton>
           <Form::SaveWithHistory
             @save={{this.saveInstance}}
+            @isSaving={{this.save.isRunning}}
```
Comment (JonasVanHoof) [inline app/components/form/instance.hbs:34]: To show the loading state of the button in the confirm modal

File: app/config/custom-inflector-rules.js [hunk 3]
```diff
@@ -71,3 +71,4 @@ inflector.irregular(
 inflector.irregular('minister-position', 'minister-positions');
 
 inflector.irregular('nationality', 'nationalities');
+inflector.irregular('bestuurseenheid-contact', 'bestuurseenheid-contact');
```
Comment (Rahien) [inline app/config/custom-inflector-rules.js:74]: would not change this to singular

File: app/controllers/settings.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+export default class SettingsController extends Controller {
+  @service currentSession;
+  @tracked bestuurseenheidEmail;
+
+  get bestuurseenheid() {
+    return { email: this.bestuurseenheidEmail };
+  }
+
+  saveBestuurseenheidSettings = restartableTask(async () => {});
```
Comment (Rahien) [inline app/controllers/settings.js:15]: this looks very strange why do we need it?

**Review by Rahien — CHANGES_REQUESTED** [review]

File: app/controllers/settings.js (hunk 4 above)
Comment (JonasVanHoof) [reply app/controllers/settings.js]: Removed it completely, didn't know that you could leav out the argument of `onSave` in form::instance

**Review by Rahien — APPROVED** [review]

**Pull request merged**